SIAB/supercomputing/test/old_version_test.py

```
@@ -179,7 +179,7 @@ def test_convert(self):
                  }
             ]
         }
-        for result in old_version.convert(calculation_setting=calculation_setting,
+        for result in ptgfront.convert(calculation_setting=calculation_setting,
```

Ruff

Loop control variable `result` not used within loop body

Rename unused `result` to `_result`

(B007)